docs/Chat/Events.md

```diff
@@ -375,413 +376,444 @@ According to Twitch, the userstate object is always subject to change.
+* `whisper_limit_per_min`: You are sending whispers too fast. Try again in a minute.
+* `whisper_limit_per_sec`: You are sending whispers too fast. Try again in a second.
+* `whisper_restricted_recipient`: That user's settings prevent them from receiving this whisper.
+
+The following msg-ids wont be returned in the `notice` event because they are already available as event listeners:
+
+* `host_off`: Exited hosting mode.
+* `host_on`: Now hosting X
+* `no_mods`: There are no moderators for this room.
+* `r9k_off`: This room is no longer in r9k mode.
+* `r9k_on`: This room is now in r9k mode.
+* `room_mods`: The moderators of this room are X
+* `slow_off`: This room is no longer in slow mode.
+* `slow_on`: This room is now in slow mode. You may send messages every X seconds.
+* `subs_off`: This room is no longer in subscribers-only mode.
+* `subs_on`: This room is now in subscribers-only mode.
+
+```javascript
+client.on('notice', function(channel, msgid, message) {
+  // Do your stuff.
+});
+```
 
 ### Part
 
 User has left a channel.
 
 **Parameters:**
 
-- ``channel``: _String_ - Channel name
-- ``username``: _String_ - Username who left the channel
-- ``self``: _Boolean_ - Client has left the channel
+* `channel`: _String_ - Channel name
+* `username`: _String_ - Username who left the channel
+* `self`: _Boolean_ - Client has left the channel
 
-~~~ javascript
-client.on("part", function (channel, username, self) {
-    // Do your stuff.
+```javascript
+client.on('part', function(channel, username, self) {
+  // Do your stuff.
 });
-~~~
+```
 
 ### Ping
 
 Received PING from server.
 
-~~~ javascript
-client.on("ping", function () {
-    // Do your stuff.
+```javascript
+client.on('ping', function() {
+  // Do your stuff.
 });
-~~~
+```
 
 ### Pong
 
 Sent a PING request ? PONG.
 
 **Parameters:**
 
-- ``latency``: _Float_ - Current latency
+* `latency`: _Float_ - Current latency
 
-~~~ javascript
-client.on("pong", function (latency) {
-    // Do your stuff.
+```javascript
+client.on('pong', function(latency) {
+  // Do your stuff.
 });
-~~~
+```
 
 ### R9kbeta
 
 Channel enabled or disabled R9K mode.
 
 **Parameters:**
 
-- ``channel``: _String_ - Channel name
-- ``enabled``: _Boolean_ - Returns ``true`` if mode is enabled or ``false`` if disabled
+* `channel`: _String_ - Channel name
+* `enabled`: _Boolean_ - Returns `true` if mode is enabled or `false` if disabled
 
-~~~ javascript
-client.on("r9kbeta", function (channel, enabled) {
-    // Do your stuff.
+```javascript
+client.on('r9kbeta', function(channel, enabled) {
+  // Do your stuff.
 });
-~~~
+```
+
+### Raid
+
+A [raid](https://help.twitch.tv/customer/portal/articles/2877820) occurred on a channel.
+
+**Parameters:**
+
+* `channel`: _String_ - Channel name
+* `raider`: _String_ - Channel raiding
+* `viewers`: _Integer_ - Number of viewers
+* `userstate`: _Object_ - Userstate
+
+```javascript
+client.on('raid', function(channel, raider, viewers, userstate) {

```

Comment:
Reflect changes to emit:
`client.on('raid', function({ channel, raider, viewers, userstate }) {`